release-notes.txt

```diff
@@ -0,0 +1,97 @@
+BISDN Linux
+v4.1.0 Release Notes
+Date: 2020-09-20
+
+! New in this release
+- The port LEDs on platform Celestica Questone 2A are now functional.
+
+! Changed behaviour
+- Due to FRR 8.0.1 now enforcing link IPv6 local addresses for various
+  IPv6 based protocols, IPv6 Link local addresses are now enabled by
+  default on all interfaces.
+- SSH over IPv6 will now also be blocked by default on non management
+  ports.
+
+! Bug fixes
+- Fixed a longstanding issue where the switch stopped forwarding
+  multicast and broadcast packets, breaking protocols like DHCP.
+- Fixed a socket leak in baseboxd, which could lead to various issues
+  after several days of uptime.
+- Fixed various issues in multicast handling in baseboxd.
```

Comment:
Drop at least one "various" in lines 18 to 20.

Reply:
replaced various issues with multiple issues.